### PR TITLE
style change

### PR DESCRIPTION
**Pull request by fan-zhang-sv**
### _Summary_

everything in the block

### CODE REVIEW

File: packages/wallet-sdk/src/util/provider.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ export function getCoinbaseInjectedSigner(): Signer | undefined {
 }
 
 function getInjectedEthereum(): CBInjectedProvider | undefined {
-  const window = globalThis as CBWindow;
   try {
+    const window = globalThis as CBWindow;
     return window.ethereum ?? window.top?.ethereum;
     // eslint-disable-next-line no-empty
   } catch {}
```
Comment (bangtoven) [inline packages/wallet-sdk/src/util/provider.ts:51]:
```suggestion
  } catch {
    return undefined;
  }
```

File: packages/wallet-sdk/src/util/provider.ts [hunk 2]
```diff
@@ -57,9 +57,8 @@ export function getCoinbaseInjectedProvider({
   metadata,
   preference,
 }: Readonly<ConstructorOptions>): ProviderInterface | undefined {
-  const window = globalThis as CBWindow;
-
   if (preference.options !== 'smartWalletOnly') {
+    const window = globalThis as CBWindow;
     const signer = getCoinbaseInjectedSigner();
```
Comment (bangtoven) [inline packages/wallet-sdk/src/util/provider.ts:62]: image

**Review by bangtoven — COMMENTED** [review]
@fan-zhang-sv

**Review by bangtoven — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for bangtoven @ 2024-06-24 17:49:52 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by bangtoven — APPROVED** [review]

**Pull request merged**